src/common/api/accounts.ts

```diff
@@ -21,6 +22,13 @@ export const fetchTransactions =
     return fetchFromSidecar(apiServer)(path);
   };
 
+export const fetchNonce =
+  (apiServer: string) =>
+  (principal: string): Promise<AddressNonces> => {
+    const path = `/address/${principal}/nonces`;
+    return fetchFromSidecar(apiServer)(path);
```

Comment:
my comments got lost during a force push

we should not create new fetchers with this `fetchFromSidecar` method, see https://github.com/blockstack/stacks-wallet-web/issues/1523

We should rely on the clients, and if this endpoint isn't support, we could create an upstream ticket to enable support.

Reply:
@aulneau My bad I had some build check failing 😞 
Yes I understand using the api is better as I mention here
https://github.com/blockstack/stacks-wallet-web/pull/1530/files#diff-2ac8b8a230a77c0902125ef70aa464b65f2868d7d0e9a75c63aaf62e45ea018eR25

But we don't know yet when this will be delivered on their side, while with this way we can have a solution that works now and we can change when the api is completed. Is there something important I am missing that the api will bring or can we ship this for now ?

Reply:
`getAccountNonces` is exposed by the latest client, you can use that in place of this method. Please rebase your branch to make use of the apiClientAtom that was introduced a few commits ago, and then you should be able to do this:
```ts
const {accountsApi} = get(apiClients);

const nonces: AddressNonces = await accountsApi.getAccountNonces();
```

Reply:
Cool ! Thanks for the info. I will update that.

Reply:
Done !